bitmap-ittybitty/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const handler = require(__dirname + '/lib/handler');
+const app = require(__dirname + '/lib/app');
+const transformer = require('./lib/transformer');
+const bitmap = __dirname +'/img/bitmap1.bmp';
+
+var bufferOne;
+var parsedBitmap;
+var newBitmap;
+
+handler.reader(bitmap, function(err, data){
+  bufferOne = data;
+  parsedBitmap = new app(bufferOne);
+  newBitmap = parsedBitmap;
```

Comment:
There isn't really a point to this step--since objects are passed by value, any changes made to newBitmap will also be made to parsedBitmap.